private-templates-service/server/package.json

```diff
@@ -24,7 +24,10 @@
     "express-validator": "^6.3.1",
     "jws": "^4.0.0",
     "passport": "^0.4.1",
-    "passport-azure-ad": "^4.2.1"
+    "passport-azure-ad": "^4.2.1",
+    "axios": "^0.19.2",
+    "mongoose": "^5.8.10",
+    "uuid": "^3.4.0"
```

Comment:
@grace These seem required for it to build properly. Even though I am linking `adaptivecards-templating-service` and also requiring it above, its dependency's are not downloaded.